recipes/imbalanced-learn/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "imbalanced-learn" %}
+{% set version = "0.2.0.dev0" %}
+package:
+  name: imbalanced-learn
+  version: {{ version }}
+
+source:
+  git_rev: {{ version }}
+  git_url: https://github.com/scikit-learn-contrib/{{ name }}.git
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
When using the `build`/`script` you can remove both `bld.bat` and `build.sh` from your recipe.